editor/import/editor_scene_importer_gltf.cpp

```diff
@@ -1083,13 +1083,6 @@ Error EditorSceneImporterGLTF::_parse_meshes(GLTFState &state) {
 				Ref<SurfaceTool> st;
 				st.instance();
 				st->create_from_triangle_arrays(array);
-				if (!p.has("targets")) {
```

Comment:
For the reference, before #30877, this line was:
```
if (p.has("targets")) {
```
which makes sense with the following comment which mention morph targets explicitly.